### DISCUSSION TITLE
"TypeError: OnnxStableDiffusionPipeline.__init__() missing 1 required positional argument: 'vae_encoder'"

### DISCUSSION THREAD

**Discussion by ChaseFlorom** [azuritecoin/OnnxDiffusersUI#2] [General]
Hey, I'm getting this error:

"TypeError: OnnxStableDiffusionPipeline.__init__() missing 1 required positional argument: 'vae_encoder'"

Earlier in the error it also says: 

"The config attributes {'vae_encoder': ['diffusers', 'OnnxRuntimeModel']} were passed to StableDiffusionOnnxPipeline, but are not expected and will be ignored. Please verify your model_index.json configuration file." 

I have followed the instructions as listed, and the test script of the txt2img works just fine. Any help appreciated, thanks!

**Comment by azuritecoin** [comment]
it looks like they updated the conversion script in between me writing the guide and you trying to setup your system. -__-

Try this version of the script instead: https://raw.githubusercontent.com/huggingface/diffusers/75bb6d2d466d742feb1e2be15f74d2605d11f0e9/scripts/convert_stable_diffusion_checkpoint_to_onnx.py

**Comment by ChaseFlorom** [comment]
Hey thanks for responding! So i deleted my folder that it created and tried the conversion again with the new script you provided...and now I'm getting that same error on the conversion instead of on opening the GUI.

**Comment by azuritecoin** [comment]
You might have a too new version of diffusers. try installing 0.5.1:  
`pip uninstall diffusers`  
`pip install diffusers==0.5.1`

**Comment by ChaseFlorom** [comment]
That did it!  Thanks for your help.

**Comment by azuritecoin** [comment]
Yay! Glad to help
